harvester/utils/utils.py

```diff
@@ -3,15 +3,14 @@
 import json
 import os
```

Comment:
Why the file name change? harvester/utils/utils.py seems a little over-nested...

Reply:
We have `utils/ckan_utils.py` as well so I wanted to group them. I'd be open to a name change suggestion, or combining the two files and then just parking them at `harvester/utils.py`. Any preferences?

Reply:
renamed to `utils/general_utils.py`